### PR TITLE
Change required fields and markdown for screenshots

### LINKED ISSUE #1306: Issue template improvement
**Descriptions:**
The `acceptance_criteria` section in the issue templates exposes some required checkboxes.
These are not meant to be checked by the issue creator. It is more content to explain to the contributor what should be done before approving the issue resolution and closing the ticket.

**Proposed solution:**
The section should be a markdown section.

### LINKED ISSUE #1340: [🐛 BUG] Screenshot in Markdown issue report
### What went wrong? 🤔

Dropping screenshots or images in the "Screenshot" section is impossible.

### Expected Behavior

It must be possible to dropdown images in the screenshot section.

### Code of Conduct

- [X] I have checked the [existing issues](https://github.com/Avaiga/taipy/issues?q=is%3Aissue+).
- [ ] I am willing to work on this issue (optional)

### PR DESCRIPTION
**Pull request by FlorianJacta**
Resolves https://github.com/Avaiga/taipy/issues/1306
Resolves https://github.com/Avaiga/taipy/issues/1340

- Screenshot field is now Markdown
- Required fields changed
- Make break lines consistent in issue forms

I created an issue to delete the No response fields in the issue as I wasn't able to find the solution:
https://github.com/Avaiga/taipy/issues/1357

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -112,9 +111,9 @@ body:
       label: Acceptance Criteria
       options:
         - label: "Ensure new code is unit tested, and check code coverage is at least 90%."
-          required: true
+          required: false
         - label: "Create a related issue in taipy-doc for documentation and Release Notes if relevant."
-          required: true
+          required: false
```
Comment (jrobinAV) [inline .github/ISSUE_TEMPLATE/bug-report.yml:127]: These lines don't represent checkboxes that the issue creator should select.
They should always be added to the issue. It is for the assignee to have the list of criteria for completing the issue.

Comment (FlorianJacta) [reply .github/ISSUE_TEMPLATE/bug-report.yml:127]: I am not sure to understand the comment

Comment (FlorianJacta) [reply .github/ISSUE_TEMPLATE/bug-report.yml:127]: Do you want me to have just the Markdown version of it?

Comment (FlorianJacta) [reply .github/ISSUE_TEMPLATE/bug-report.yml:127]: I have changed it; tell me if this is what you meant

Comment (FabienLelaquais) [reply .github/ISSUE_TEMPLATE/bug-report.yml:127]: The code you have changed would seem to fix that problem.
But I can only confirm when I see that in action.

**Review by FabienLelaquais — APPROVED** [review]
Well done!

**Review by jrobinAV — APPROVED** [review]

**Pull request merged**